NEWS.md

```diff
@@ -11,6 +11,14 @@
 
 * Failure locations are now formated as R error locations.
 
+* The [Catch](https://github.com/philsquared/Catch) unit
+  testing library can now be used for unit testing of C++
+  files. You can place C++ files within the `tests/testthat/cpp`
+  folder named as `test-*.cpp`, and `test_dir()` and friends
+  will automatically compile and run tested examples. See
+  `system.file(package = "testthat", "resources", "test-example.cpp")`
```

Comment:
For the sake of testing (and code coverage!) it would be useful to include an example